### PR TITLE
CU-284cuwe Basic audio streaming of adaptive bitrate files from s3

### PR DESCRIPTION
**Pull request by andymiller813**

### CODE REVIEW

**Comment by newmskywalker** [comment]
Task linked: [CU-2e0d3av Clean up gradle scripts by creating helper scripts](https://app.clickup.com/t/2e0d3av)

File: android/feature-now-playing/src/main/java/io/projectnewm/feature/now/playing/NowPlayingScreen.kt [hunk 1]
```diff
@@ -70,6 +91,12 @@ fun NowPlayingScreen() {
         Spacer(modifier = Modifier.height(24.dp))
 
         TipIcon()
+
+        if (firstLoad.value) {
+            Player.play(context)
+            firstLoad.value = false
+            isPlaying.value = true
+        }
```
Comment (newmskywalker) [inline android/feature-now-playing/src/main/java/io/projectnewm/feature/now/playing/NowPlayingScreen.kt]: Could `LaunchedEffect(Unit) { Player.play(context) }` potentially replace the `firstLoad` state?

**Review by newmskywalker — APPROVED** [review]
Looks good! Just a minor comment about potentially using `LaunchedEffect`

File: android/feature-now-playing/src/main/java/io/projectnewm/feature/now/playing/NowPlayingScreen.kt (hunk 1 above)
Comment (andymiller813) [reply android/feature-now-playing/src/main/java/io/projectnewm/feature/now/playing/NowPlayingScreen.kt]: Updated, I think that works

**Pull request merged**